docs/resources/gno-interrealm.md

```diff
@@ -440,16 +440,17 @@ func init() {
 
     // Returns (
     //     addr:g1user,
-    //     pkgpath:"gno.land/r/g1user/run"
-    // ) == std.NewCodeRealm("gno.land/r/g1user/run")
+    //     pkgpath:"gno.land/e/g1user/run"
+    // ) == std.NewCodeRealm("gno.land/e/g1user/run")
     std.CurrentRealm()
 }
 ```
 
 The same applies for p package initialization. Initialization and tests are the
 only times that `std.CurrentRealm()` will return a p package path that starts
 with "/p/" instead of "/r/". The package is technically still mutable during
-initialization.
+initialization. Note that the /e/ namespace is used for run paths (ephemeral),
+while /r/ is used for normal realms.
```

Comment:
This part is confusing to me, I don't understand the relation with this new line and the reste of the paragraph which explains that the `std.CurrentRealm()` can return a `/p/` path only if called from the `init()` of a pure package.